### ISSUE TITLE
That escalated quickly! Prototyping IoT with JavaScript

### ISSUE THREAD

**Issue by eiriksm** [nodeconf/budapest#18]
#### Title

That escalated quickly! Prototyping IoT with JavaScript
#### Description

Let me tell you a story. Or wait, 2 stories.

One of the stories is about a man who wanted pretty graphs and useful logs about different aspects of his beer brewing process. He decided to use JavaScript. Before he knew what he got himself into, he had made graphs of temperature in the mashing, boiling and fermenting process. That was when it escalated. The story speaks of logging of plops per minute (ppm™) and real-time streaming of Co2 levels turned into midi tones.

The other story is about a man who had a slot car racing track. He wanted to control this programatically. He decided to use JavaScript. Before he knew it, a press of a button on a webpage ran the race car 2 rounds on the track (including loops). That was when it escalated. The story speaks of webRTC streaming of a slot car "dashcam", car speed measurement scoring systems and VR with Google Cardboard. Escalated indeed.
#### Learning objectives

This session will not highlight any new CS paradigms. There will be less “wow, that was useful” and more “Why did he do that? Because he could”. The “transpilation example to GIF ratio” (tetg ratio™ for short) will probably be in favor of the image format. And since I already promised beer, slot cars, GIFs and VR, I will also promise both pictures of cats and dogs.

In short, the session is about enthusiasm for JavaScript, hacking together things, GIFs and life itself.
#### City of residence

Trondheim

**Comment by gergelyke** [comment]
Hey there!

We really, really thank you for your participation in the Call for Proposal for NodeConf Budapest 2017!

We're sorry to tell you, that although your topic was really great, it didn't get accepted to our event this year. It wasn't an easy decision, and it took a while for us, but with all the factors that influenced us during the selection process, it had to be made.

To compensate this sad news a bit, we would like to invite you to come as an attendee! As an appreciation for the effort you already made to make this event great, we'd like to give you a ticket with a 20% discount: 
https://ti.to/risingstack/nodeconf-budapest/discount/thanks-for-your-cfp

We hope you can make it!
